notebooks/text_classification/text_classification_classical_text_explainer.ipynb

```diff
@@ -37,21 +38,21 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "scrolled": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
+    "import sys\n",
+    "sys.path.append(\"../..\")\n",
     "import os\n",
-    "from interpret_text.classical.classical_text_explainer import ClassicalTextExplainer\n",
     "\n",
     "# sklearn\n",
     "from sklearn.metrics import precision_recall_fscore_support\n",
     "from sklearn.model_selection import train_test_split\n",
     "from sklearn.preprocessing import LabelEncoder\n",
     "\n",
-    "# nlp recipes\n",
-    "from interpret_text.common.utils_unified import load_pandas_df\n",
+    "from interpret_text.classical.classical_text_explainer import ClassicalTextExplainer\n",
```

Comment:
We should add ClassicalTextExplainer to interpret_text/classical/__init__.py. Then the import would be
from interpret_text.classical import ClassicalTextExplainer